script.js

```diff
@@ -0,0 +1,171 @@
+    technologies: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    featureImg: './images/Snapshoot.svg',
+    liveLink: '#pasteLink',
+    linkSource: 'githubLink',
+    iconCancel: './images/Cancel.svg',
+  },
+  {
+    title: 'Keeping track of hundreds  of components website',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    technologies: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    featureImg: './images/Snapshoot.svg',
+    liveLink: '#pasteLink',
+    linkSource: 'githubLink',
+    iconCancel: './images/Cancel.svg',
+  },
+  {
+    title: 'Keeping track of hundreds  of components website',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    technologies: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    featureImg: './images/Snapshoot.svg',
+    liveLink: '#pasteLink',
+    linkSource: 'githubLink',
+    iconCancel: './images/Cancel.svg',
+  },
+];
+const seeProject = document.querySelectorAll('.popup');
+const section = document.createElement('div');
+section.className = 'popup-container';
+let content = '';
+data.forEach((item) => {
+  content = `
+  <div class="popup-details-container">
+    <span class="popup-name-image">
+      <h1 class="popup-name">${item.title}</h1>
+      <img src=${item.iconCancel} class="popup-image"> 
+    </span>
+    <ul class="popup-unorded-list">
+      <li class="popup-list">${item.technologies[0]}</li>
+      <li class="popup-list">${item.technologies[1]}</li>
+      <li class="popup-list">${item.technologies[2]}</li>
+    </ul>
+    <span class="desktop-diff">
+      <img src=${item.featureImg} class="popup-feature-img">
+      <div class="left-side">
+        <p class="popup-description">${item.description}</p>
+        <span class="popup-button">
+          <button class="button popup-button-1">
+            <a href=${item.liveLink}>See Project</a>
+            <img src="./images/see live.svg">
+          </button>
+          <button class="button popup-button-1">
+            <a href=${item.linkSource}>See Source</a>
```

Comment:
<img width="441" alt="3" src="https://user-images.githubusercontent.com/77843050/225448561-42a096b1-3f77-4bc3-95fa-1cd189f86d28.png">

- [ ] Fix this space issue, take a look at the original design and adjust yours